java/native-image/java-native-image-sample/README.md

```diff
@@ -7,15 +7,13 @@
 ```bash
 pack build applications/native-image \
   --builder paketobuildpacks/builder-jammy-tiny \
-  --env BP_NATIVE_IMAGE=true \
-  --env BP_MAVEN_BUILD_ARGUMENTS="-Dmaven.test.skip=true --no-transfer-progress package -Pnative" \
-  --env BP_JVM_VERSION=17
+  --env BP_MAVEN_ACTIVE_PROFILES=native \
 ```
 
 ### With the Spring Boot Maven Plugin
 
 ```bash
-./mvnw -Dmaven.test.skip=true spring-boot:build-image -Pnative
+./mvnw spring-boot:build-image -Pnative
```

Comment:
My understanding is that this change requires Spring Boot 3.1.0+, right? If that's true, I think we should keep the previous example for Spring Boot 3.0, at least for a little while longer. It's still in support through 2023-11-24.

Reply:
ah, makes sense